notebooks/kubeflow_pipelines/walkthrough/labs/kfp_walkthrough_vertex.ipynb

```diff
@@ -28,6 +28,7 @@
     "\n",
```

Comment:
<p>Fix grammar: "Complete the&nbsp;<code>HyperparameterTuningJob</code>&nbsp;command below to configure the hyperparameter tuning engine with the following parameter values:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/702/discussion/#comment-2430742752'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>